Raylib-cs/types/Window.cs

```diff
@@ -0,0 +1,168 @@
+    
+    /// <summary> See <see cref="Raylib.SetWindowSize"/> </summary>
+    public void SetSize(int width, int height) => Raylib.SetWindowSize(width, height);
+    
+    /// <summary> See <see cref="Raylib.SetWindowOpacity"/> </summary>
+    public void SetOpacity(float opacity) => Raylib.SetWindowOpacity(opacity);
+    
+    /// <summary> See <see cref="Raylib.EnableEventWaiting"/> </summary>
+    public void EnableEventWaiting() => Raylib.EnableEventWaiting();
+    
+    /// <summary> See <see cref="Raylib.DisableEventWaiting"/> </summary>
+    public void DisableEventWaiting() => Raylib.DisableEventWaiting();
+    
+    
+    /// <summary> See <see cref="Raylib.GetScreenWidth"/> </summary>
+    public int GetScreenWidth() => Raylib.GetScreenWidth();
+    
+    /// <summary> See <see cref="Raylib.GetScreenHeight"/> </summary>
+    public int GetScreenHeight() => Raylib.GetScreenHeight();
+    
+    /// <summary> See <see cref="Raylib.GetRenderWidth"/> </summary>
+    public int GetRenderWidth() => Raylib.GetRenderWidth();
+    
+    /// <summary> See <see cref="Raylib.GetRenderHeight"/> </summary>
+    public int GetRenderHeight() => Raylib.GetRenderHeight();
+    
+    /// <summary> See <see cref="Raylib.GetMonitorCount"/> </summary>
+    public int GetMonitorCount() => Raylib.GetMonitorCount();
+    
+    /// <summary> See <see cref="Raylib.GetCurrentMonitor"/> </summary>
+    public int GetCurrentMonitor() => Raylib.GetCurrentMonitor();
+    
+    /// <summary> See <see cref="Raylib.GetMonitorPosition"/> </summary>
+    public Vector2 GetMonitorPosition(int monitor) => Raylib.GetMonitorPosition(monitor);
+    
+    /// <summary> See <see cref="Raylib.GetMonitorWidth"/> </summary>
+    public int GetMonitorWidth(int monitor) => Raylib.GetMonitorWidth(monitor);
+    
+    /// <summary> See <see cref="Raylib.GetMonitorHeight"/> </summary>
+    public int GetMonitorHeight(int monitor) => Raylib.GetMonitorHeight(monitor);
+    
+    /// <summary> See <see cref="Raylib.GetMonitorPhysicalWidth"/> </summary>
+    public int GetMonitorPhysicalWidth(int monitor) => Raylib.GetMonitorPhysicalWidth(monitor);
+    
+    /// <summary> See <see cref="Raylib.GetMonitorPhysicalHeight"/> </summary>
+    public int GetMonitorPhysicalHeight(int monitor) => Raylib.GetMonitorPhysicalHeight(monitor);
+    
+    /// <summary> See <see cref="Raylib.GetMonitorRefreshRate"/> </summary>
+    public int GetMonitorRefreshRate(int monitor) => Raylib.GetMonitorRefreshRate(monitor);
+    
+    /// <summary> See <see cref="Raylib.GetWindowPosition"/> </summary>
+    public Vector2 GetPosition() => Raylib.GetWindowPosition();
+    
+    /// <summary> See <see cref="Raylib.GetWindowScaleDPI"/> </summary>
+    public Vector2 GetScaleDpi() => Raylib.GetWindowScaleDPI();
+    
+    /// <summary> See <see cref="Raylib.GetMonitorName_"/> </summary>
+    public string GetMonitorName(int monitor) => Raylib.GetMonitorName_(monitor);
+
+    public void Dispose()
```

Comment:
I do not fully understand, would it not work like that to?
```cs
using var window = Window.Instance(ConfigFlags.FLAG_WINDOW_RESIZABLE, 800, 600, "My Window");
```

Sry if i missunderstand something, but is it not like that it work?

Reply:
No thats right. Its just an extra precaution in case they write this instead:
```c#
var window = Window.Instance(ConfigFlags.FLAG_WINDOW_RESIZABLE, 800, 600, "My Window");
```
The idea with the finalizer is that the gc will call it as the app is closeing.
You could write it so it looks like this:
```c#
~Window() => Dispose();
```

Reply:
Finalizes do not work in dotnet core or dotnet 5+. Do not use them.

Only use IDisposable.

Reply:
> Finalizes do not work in dotnet core or dotnet 5+. Do not use them.
> 
> Only use IDisposable.
> 
> [dotnet/csharpstandard#291](https://github.com/dotnet/csharpstandard/issues/291)

Noted, this could be added later anyways